CONTRIBUTING.md

```diff
@@ -25,6 +25,41 @@ Entries should be listed in alphabetical order. Edit other yaml files as needed.
 1. Follow the instructions included in the news template and name the new file after the title of the news article. 
 1. Entries will be listed in chronological order automatically.
 
+### <a name="commit-signing">Signing-off on Commits (Developer Certificate of Origin)</a>
+
+To contribute to this project, you must agree to the Developer Certificate of
+Origin (DCO) for each commit you make. The DCO is a simple statement that you,
+as a contributor, have the legal right to make the contribution.
+
+See the [DCO](https://developercertificate.org) file for the full text of what you must agree to
+and how it works [here](https://github.com/probot/dco#how-it-works).
+To signify that you agree to the DCO for contributions, you simply add a line to each of your
+git commit messages:
+
+```
+Signed-off-by: Jane Smith <jane.smith@example.com>
+```
+
+In most cases, you can add this signoff to your commit automatically with the
+`-s` or `--signoff` flag to `git commit`. You must use your real name and a reachable email
+address (sorry, no pseudonyms or anonymous contributions). An example of signing off on a commit:
+```
+$ commit -s -m “my commit message w/signoff”
+```
+
+To ensure all your commits are signed, you may choose to add this alias to your global ```.gitconfig```:
+
+*~/.gitconfig*
+```
+[alias]
+  amend = commit -s --amend
+  cm = commit -s -m
+  commit = commit -s
+```
+Or you may configure your IDE, for example, Visual Studio Code to automatically sign-off commits for you:
+
+<a href="https://user-images.githubusercontent.com/7570704/64490167-98906400-d25a-11e9-8b8a-5f465b854d49.png" ><img src="https://user-images.githubusercontent.com/7570704/64490167-98906400-d25a-11e9-8b8a-5f465b854d49.png" width="50%"><a>
```

Comment:
@HarshKapadia2 nice work!

While what you have here will do the trick, I wonder if might refactor this to keep a copy of the screenshots local this to repo. Remote dependencies have a way of moving without notice and we end up with broken links. 

One of these issues identifies which folder to put those screenshots (images) into. 👍

Reply:
@ashiscs FYI

Reply:
Oh okay.
I'll add the screenshot to `./assets/images`.

Reply:
Thank you, sir. ✔️

Reply:
I made the changes in PR #474 (I am a little inexperienced). I'm sorry if that inconveniences you.

Reply:
1. On your local machine go to the branch (HarshKapadia2:pull_request_template) do the required changes which were suggested. [for changing branch = git checkout HarshKapadia2:pull_request_template]
2. Save the file.
3. Commit and push, it will be automatically reflected in these pr.
Hope it helps.

Reply:
I got it, but now that I have made a separate PR, do you need me to make a PR here?
Lmk, I'll do it.

Reply:
For now, we can go with these pr  !